src/features/user/TreeMapper/Analytics/components/SpeciesPlanted/index.tsx

```diff
@@ -36,7 +33,15 @@ export const SpeciesPlanted = () => {
 
   const { project, fromDate, toDate } = useAnalytics();
 
-  const { handleError } = useContext(ErrorHandlingContext);
+  const { makeRequest } = useNextRequest<{ data: ISpeciesPlanted[] }>(
+    '/api/data-explorer/species-planted',
+    HTTP_METHOD.POST,
+    {
+      projectId: project?.id,
+      startDate: fromDate,
+      endDate: toDate,
+    }
+  );
```

Comment:
Even though there isn't a scope of conflict here, adding an alias for `makeRequest` here would make the `fetchPlantedSpecies` function a little more readable.

Reply:
@mohitb35 So the idea behind `useNextRequest` hook is to make a single component responsible of making a request thus, the function name `makeRequest`. Though there could be an exception where we can use it more than once which could result in conflicts so using alias there make sense but not in this case.